alarm-frontend/src/Components/Alarm.js

```diff
@@ -1,63 +1,67 @@
-import React, { useContext } from "react";
-import { minutesNumber, hourNumber } from "../fixNumbers";
-import useOption from "../Hooks/useOption";
+import React, { useContext, useEffect, useState } from "react";
 import { Context } from "./Context";
+import '../css/alarm.css';
+import Sound from "../LostWoods.mp3";
+import axios from "axios";
+const BASE_URL = "http://127.0.0.1:5000"
 
-function Alarm() {
-    const [hour, setHour] = useOption("Hour");
-    const [minutes, setMinutes] = useOption("Minutes");
-    const [amPmOption, setAmPmOption] = useOption("Am-Pm");
-    const { setAlarmTime, pauseAlarm, alarmExists, setAlarmExists } =
-        useContext(Context);
-
+function Alarm({AMorPM, difficulty, hour, id, minute, type, handler}) {
+    const [alarm] = useState(new Audio(Sound))
```

Comment:
This looks slightly sketch for a few reasons:
- Creating a new Audio object during each render is inefficient
- Since you're creating a new Audio object during the render, you might also want to consider cleaning it up when the component unmounts. Otherwise, you might have multiple Audio objects playing at the same time if the component is re-rendered multiple times.

This might work:
```
import { useEffect, useRef } from "react";

const MyComponent = () => {
  const audioRef = useRef(null);

  useEffect(() => {
    audioRef.current = new Audio(Sound);

    return () => {
      if (audioRef.current) {
        audioRef.current.pause();
        audioRef.current = null;
      }
    };
  }, [Sound]);

  const playAlarm = () => {
    if (audioRef.current) {
      audioRef.current.play();
    }
  };

  return (
    <div>
      {/* Render your component */}
      <button onClick={playAlarm}>Play Alarm</button>
    </div>
  );
};

```